packages/web/pages/search/guilds.tsx

```diff
@@ -0,0 +1,33 @@
+import { PageContainer } from 'components/Container';
+import { GuildList } from 'components/Guild/GuildList';
+import SearchFilters from 'components/SearchFilters';
+import { HeadComponent } from 'components/Seo';
+import { GuildFragment } from 'graphql/autogen/types';
+import { searchGuilds } from 'graphql/queries/guild';
+import { useRouter } from 'next/router';
+import React, { useEffect, useState } from 'react';
+import { GlobalFilters } from 'utils/GlobalSearch';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const GuildSearchPage = () => {
```

Comment:
```suggestion
const GuildSearchPage: React.FC = () => {
```

then we can remove the eslint-disable